stock_move_location/wizard/stock_move_location.py

```diff
@@ -130,8 +130,6 @@ def action_move_location(self):
         picking = self._create_picking()
         self._create_moves(picking)
         if not self.env.context.get("planned"):
-            picking.action_confirm()
-            picking.action_assign()
```

Comment:
I think assign action is still needed

Reply:
Why?

Reply:
Because being immediate transfer doesn't grant you that you have enough available stock.

Reply:
If you re-check availability the lots informed in wizard lines can be recreate with other ones...
In wizar lines already check availability quantity.
https://github.com/OCA/stock-logistics-warehouse/blob/c9af30a89e6efb339655a1d2a79edcc0c39c6a88/stock_move_location/wizard/stock_move_location_line.py#L110

Reply:
OK, got it. Thanks.